RecoPixelVertexing/PixelTriplets/plugins/CAGraph.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * CAGraph.h
+ *
+ *  Created on: Aug 19, 2016
+ *      Author: fpantale
+ */
+
+#ifndef CAGRAPH_H_
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h